contracts/interfaces/IbcReceiverUpgradeable.sol

```diff
@@ -57,4 +56,12 @@ contract IbcReceiverBaseUpgradeable is OwnableUpgradeable {
         __Ownable_init();
         dispatcher = _dispatcher;
     }
+
+    /**
+     * @dev This empty reserved space is put in place to allow future versions to add new
+     * variables without shifting down storage in the inheritance chain.
+     * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
+     */
+    // solhint-disable-next-line ordering
+    uint256[49] private __gap;
```

Comment:
ok so the gap was moved from the other place to here?

Reply:
exactly

Reply:
the reason why it's better in the IbcreceiverUpgradeable is that any contracts which inherit from it (e.g. user Dapps) are also now going to be upgrade-safe in case we decide we want to add any storage vars to this contract